sources/editor/Stride.Assets.Presentation/AssetEditors/ProjectWatcher.cs

```diff
@@ -23,6 +21,11 @@
 using Stride.Assets.Presentation.AssetEditors.ScriptEditor;
 using Project = Microsoft.CodeAnalysis.Project;
 using System.Diagnostics;
+using System.Linq;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.Build.Execution;
+using System.Diagnostics.CodeAnalysis;
+using QuickGraph;
```

Comment:
Although it is already messy in this file, try to keep the usings sorted.

Reply:
https://github.com/stride3d/stride/pull/2106/commits/ca151ad67bdbb4317d569469fba98d795f2b1f92
refactored it